keps/sig-cli/1440-kubectl-events/README.md

```diff
@@ -138,17 +143,41 @@ Additionally, the new command should support all the printing flags available in
 
 ### Test Plan
 
-In addition to standard unit tests for kubectl, the events command will be released as a kubectl alpha subcommand, signaling users to expect instability. During the alpha phase we will gather feedback from users that we expect will improve the design of debug and identify the Critical User Journeys we should test prior to Alpha -> Beta graduation.
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+Before any additional functional updates we need to ensure the current functionality
+is properly cover with unit and integration (`test/cmd`) tests.
+Before promoting to beta at least a single e2e test should also be added in
+`k8s.io/kubernetes/test/e2e/kubectl/kubectl.go`.
+
+##### Unit tests
+
+- `k8s.io/kubectl/pkg/cmd/events`: `2022-06-09` - `0%`
+
+##### Integration tests
+
+- missing
+
+##### e2e tests
+
+- missing
```

Comment:
All of that are a bit worrying.

Can we first fix that and add tests and only then get back to applying the promotion to Beta?

This thread is related:
https://kubernetes.slack.com/archives/C2C40FMNF/p1654710138987519
and we should treat everyone the same way :)

Reply:
Yeah, that's fair, I'll back it out for now. We'll add on tests and stabilization as is, and promote to beta in 1.26